sdk/appconfiguration/Azure.Data.AppConfiguration/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,165 @@
+  Content-Length: 0
+}
+```
+
+### Enable client logging
+
+To troubleshoot issues with the library, first enable logging to monitor the behavior of the application. The errors and warnings in the logs generally provide useful insights into what went wrong and sometimes include corrective actions to fix issues.
+
+This library uses the standard [logging](https://docs.microsoft.com/dotnet/azure/sdk/logging) library. Basic information about HTTP sessions, such as URLs and headers, is logged at the `INFO` level.
+
+The simplest way to see the logs is to enable console logging. To create an Azure SDK log listener that outputs messages to the console, use the [AzureEventSourceListener.CreateConsoleLogger](https://docs.microsoft.com/dotnet/api/azure.core.diagnostics.azureeventsourcelistener.createconsolelogger?view=azure-dotnet) method:
+
+```csharp
+using Azure.Core.Diagnostics;
+
+// set up a listener to monitor logged events
+using AzureEventSourceListener listener = AzureEventSourceListener.CreateConsoleLogger();
+```
+
+#### Enable content logging
+
+By default only URI and headers are logged. To enable content logging, set the `Diagnostics.IsLoggingContentEnabled` client option. 
+
+``` c#
+ConfigurationClientOptions options = new ConfigurationClientOptions()
+{
+    Diagnostics =
+    {
+        IsLoggingContentEnabled = true
+    }
+};
+```
+#### Logging redacted headers and query parameters
+
+Some sensitive headers and query parameters are not logged by default and are displayed as "REDACTED", to include them in logs use the `Diagnostics.LoggedHeaderNames` and `Diagnostics.LoggedQueryParameters` client options.
+
+``` c#
+ConfigurationClientOptions options = new ConfigurationClientOptions()
+{
+    Diagnostics =
+    {
+        LoggedHeaderNames = { "x-ms-request-id" },
+        LoggedQueryParameters = { "api-version" }
+    }
+};
+```
+
+You can also disable redaction completely by adding a "*" to collections mentioned above.
+
+``` c#
+ConfigurationClientOptions options = new ConfigurationClientOptions()
+{
+    Diagnostics =
+    {
+        LoggedHeaderNames = { "*" },
+        LoggedQueryParameters = { "*" }
+    }
+};
+```
+
+To learn about other logging mechanisms, see [Azure SDK diagnostics](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/Diagnostics.md).
+
+## Troubleshooting authentication issues
+
+Azure App Configuration supports Azure Active Directory authentication. To provide a valid credential, you can use the `Azure.Identity` package. For more information on getting started, see the [Azure App Configuration library's README](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/appconfiguration/Azure.Data.AppConfiguration#authenticate-the-client). For details on the credential types supported in `Azure.Identity`, see the [Azure Identity library's documentation](https://docs.microsoft.com/dotnet/api/overview/azure/Identity-readme).
```

Comment:
Let's shift this to the "overview" on MS Docs.   Same README, same content, just less GitHub-y.

https://learn.microsoft.com/dotnet/api/overview/azure/data.appconfiguration-readme